recipes/libtiledb-sql-py/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "libtiledb-sql-py" %}
+{% set version = "2.0.0" %}
+{% set sha256 = "c0228d64bdf9eeabdac5284c760bf4899892a170f769688a09285bcbd46963d1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/TileDB-Inc/TileDB-SQL-Py.git
+  git_tag: {{ version }}
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+  skip: True  # [py2k]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - wheel
+    - setuptools
+    - setuptools_scm
+    - libtiledb-sql
+  run:
+    - python
+    - setuptools
+    - libtiledb-sql
+
+test:
+  imports:
+    - tiledb.sql
+  commands:
+    - python -c "import tiledb.sql"
+
+about:
+  home: https://tiledb.com
+  license: GPLv2
+  license_family: GPL
+  license_file: LICENSE
+  summary: 'libtiledb-sql-py is a Embedded Python SQL interface for TileDB arrays using the MyTile storage engine'
+  description: |
+    libtiledb-sql-py is a fork of the mysqlclient-python which builds against libtiledb-sql to provide embedded sql querying of TileDB arrays.\n
+
+    Docs: https://docs.tiledb.com/developer/api-usage/embedded-sql
+
+    Source: https://github.com/TileDB-Inc/TileDB-SQL-Py
+
+    Upstream source: https://github.com/PyMySQL/mysqlclient-python
+
+  doc_url: https://docs.tiledb.com/developer/api-usage/embedded-sql
+  dev_url: https://github.com/TileDB-Inc/TileDB-SQL-Py
+
+extra:
+  recipe-maintainers:
+    - shelnutt2
+    - ihnorton
```

Comment:
@ihnorton are you OK with being a maintainer?

Reply:
yes